examples/example_plugin/extended-veiwer.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery'
+
+export function ExtendedVeiwer(iv) {
+    this._iv = iv;
+}
+
+const highlightToggle = "red-highlight-toggle-on";
+const highlight = "example-containing-t-red-highlight";
+
+ExtendedVeiwer.prototype.extendDisplayOptionsMenu = function(menu) {
+    let iv = this._iv;
+    menu.addCheckboxItem("Example Menu Item (Highlight Words containing the letter 'T' Red)", function(checked) {
```

Comment:
If this used an arrow function (`(checked) => { ...  }`), I think the `iv` variable wouldn't be needed.